### PR TITLE
support Bootstrap 4

### LINKED ISSUE #39: compatibility with Bootstrap 4?
I'm trying this out on a page which uses Bootstrap 4. No luck yet: the menu aligns horizontally and it scrolls off the top of the page.

Bootstrap [mentions](https://v4-alpha.getbootstrap.com/migration/) that `affix` is no longer supported in v4. 

Do you already see any easy workarounds? (or maybe my symptoms just suggest user error?)

Thanks--

### PR DESCRIPTION
**Pull request by afeld**
Closes #39.

* [x] Upgrade Bootstrap in demo site
* [x] Switch from using Affix component ([support was dropped](https://v4-alpha.getbootstrap.com/migration/#components))
* [x] Fix ScrollSpy
* [ ] ~~Ensure backwards-compatibility (create a separate test page for Bootstrap 3)~~

### CODE REVIEW

**Comment by alcalyn** [comment]
Links in navigation was inline, so I needed to add:

``` css
nav[data-toggle='toc'] .nav {
  flex-direction: column;
}
```

**Comment by lokimckay** [comment]
Completed steps 1-3 through path of least resistance
PR: #43

File: _layouts/default.html [hunk 1]
```diff
@@ -8,12 +8,13 @@
...
     <link rel="stylesheet" href="{{ site.baseurl }}/bootstrap-toc.css" media="screen" charset="utf-8">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/8.9.1/styles/github.min.css">
     <link rel="stylesheet" href="{{ site.baseurl }}/assets/screen.css" media="screen" charset="utf-8">
-    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js" integrity="sha512-K1qjQ+NcF2TYO/eI3M6v8EiNYZfA95pQumfvcVrTHtwQVDG+aHRqLi/ETn2uB+1JqwYqVG3LIvdm9lj6imS/pQ==" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js" integrity="sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4" crossorigin="anonymous"></script>
```
Comment (afeld) [inline _layouts/default.html]: Are we actually using this?

Comment (lokimckay) [reply _layouts/default.html]: - Have not used popper personally, but from a quick google it doesn't look like it is being used anywhere
- jQuery is definitely being used

**Comment by afeld** [comment]
I think this is good to go. @sneyed Mind giving it another look, just to make sure I didn't miss anything?

**Comment by lokimckay** [comment]
Looks great to me except for one issue: when sticking, there is no gap between the top and toc (see image)
https://user-images.githubusercontent.com/14341953/37190429-60e93bc2-23a5-11e8-9c65-59c8bc65cc35.png

Are you happy with this? I am thinking we should add a small amount of margin or padding - not sure which.

**Comment by RishikeshDarandale** [comment]
@afeld, when this will be available as a release version?

**Comment by afeld** [comment]
As soon as I can get back to it and figure out why there's a discrepancy in what @sneyed and I are seeing.

**Comment by RishikeshDarandale** [comment]
Okay. `.stick-top` has `top` as 0, thus there is no gap. Probably below should work:
```
.sticky-top {
  top: 42px;
}
```

**Comment by afeld** [comment]
Ok, fixed, and changed the site to share styles between the documentation and the site itself. Mind taking another look?

**Comment by RishikeshDarandale** [comment]
Thanks! Looks great! For reference check [here](https://rishikeshdarandale.github.io/aws-http/).

**Comment by lokimckay** [comment]
Thanks Aidan, great work :+1:

**Comment by afeld** [comment]
Released as v1.0.0. Thanks everyone!

**Pull request merged**